pkg/networkmanager/interfaces.js

```diff
@@ -1701,7 +1701,7 @@
                                {
                                    autoconnect: true,
                                    interface_name: iface.Name,
-                                   member_type: member_type,
+                                   member_type,
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18417-20230227-210817-602a6f78-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)